src/storage/mvcc/txn.rs

```diff
@@ -9,7 +9,7 @@ use kvproto::kvrpcpb::LockInfo;
 use txn_types::{Key, Lock, PessimisticLock, TimeStamp, Value};
 
 use super::metrics::{GC_DELETE_VERSIONS_HISTOGRAM, MVCC_VERSIONS_HISTOGRAM};
-use crate::storage::{kv::Modify, mvcc::PessimisticLockNotFoundReason};
+use crate::storage::kv::Modify;
```

Comment:
I see the `PessimisticLockNotFoundReason` is used on the function `make_txn_error`, but the compile warning is false positive. Maybe better to move this import into `make_txn_error`

Reply:
done